src/main/js/controls/Graph/helpers/helpers.js

```diff
@@ -75,18 +76,45 @@ const getAxisInfoRowLabelHeight = (config) =>
  * @private
  * @param {object} config - config object derived from input JSON
  * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
- * @returns {object} d3 svg path
+ * @returns {undefined} d3 svg path
```

Comment:
Invalid return description

Reply:
what will be correct return for this i am not sure.

Reply:
`@returns {undefined} - returns nothing`